business/workloads.go

```diff
@@ -5,6 +5,7 @@ import (
 	"sync"
 	"time"
 
+	"github.com/kiali/k-charted/model"
```

Comment:
Perhaps an alias "kmodel" can help to not get confused between "model" (from k-charted) and "models" (from kiali).